assets/public/css/styles.css

```diff
@@ -0,0 +1,304 @@
+@font-face {
+  font-family: "axis";
+  src: url("../fonts/axis.otf");
+}
+@font-face {
+  font-family: "Montserrat-Bold";
+  src: url("../fonts/Montserrat-Bold.ttf");
+}
+@font-face {
+  font-family: "Montserrat-ExtraBold";
+  src: url("../fonts/Montserrat-ExtraBold.ttf");
+}
+* {
```

Comment:
try to avoid using "Select all"  selector

Reply:
These are defined in _resets.scss and are meant to remove / override default browser styles that don't match the design styling. Otherwise I would be writing redundant code again and again in all places to get the same styling property values.